modules/mcg-about-multicloud-gitops.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: CONCEPT
+:imagesdir: ../../images
+
+[id="about-multicloud-gitops-pattern"]
+= About the Multicloud GitOps pattern
+//AI:needs SME review
+
+//AI self: Note that Description list does not render correctly on the site
+Use case::
+
+* Use a GitOps approach to manage hybrid and multi-cloud deployments across both public and private clouds.
+* Enable cross-cluster governance and application lifecycle management.
+* Securely manage secrets across the deployment.
++
+[NOTE]
+====
+Based on the requirements of a specific implementation, certain details might differ. However, all validated patterns that are based on a portfolio architecture, generalize one or more successful deployments of a use case.
+====
+
+Background::
+Organizations are aiming to develop, deploy, and operate applications on an open hybrid cloud in a stable, simple, and secure way. This hybrid strategy includes multi-cloud deployments where workloads might be running on multiple clusters and on multiple clouds- private or public.
```

Comment:
"and on multiple clouds-"

Maybe a comma instead of a dash. e.g. "and on multiple clouds, private or public."